### PR TITLE
remove direct access of cordova methods

### PR DESCRIPTION
**Pull request by dkanada**
There were some methods getting called directly which shoudn't really happen because it means we have to modify web code more frequently than required to change app behavior.

### CODE REVIEW

File: src/scripts/site.js [hunk 1]
```diff
@@ -870,8 +870,20 @@ var Dashboard = {
...
+        var list = [];
+        list.push("bower_components/emby-webcomponents/playback/playbackvalidation");
+        list.push("bower_components/emby-webcomponents/playback/playaccessvalidation");
+        list.push("bower_components/emby-webcomponents/playback/experimentalwarnings");
+        list.push("bower_components/emby-webcomponents/htmlaudioplayer/plugin");
+        list.push("bower_components/emby-webcomponents/htmlvideoplayer/plugin");
+        list.push("bower_components/emby-webcomponents/photoplayer/plugin");
+        list.push("bower_components/emby-webcomponents/youtubeplayer/plugin");
+        "cordova" === self.appMode && list.push("cordova/chromecast");
```
Comment (JustAMan) [inline src/scripts/site.js]:
```suggestion
        if ("cordova" === self.appMode) {
            list.push("cordova/chromecast");
        }
```

File: src/scripts/site.js [hunk 2]
```diff
@@ -870,8 +870,20 @@ var Dashboard = {
...
+        "android" === self.appMode && list.push("cordova/externalplayer");
```
Comment (JustAMan) [inline src/scripts/site.js]:
```suggestion
        if ("android" === self.appMode) {
            list.push("cordova/externalplayer");
        }
```

File: src/scripts/site.js [hunk 3]
```diff
@@ -870,8 +870,20 @@ var Dashboard = {
...
+        if (appHost.supports("remotecontrol")) {
+            list.push("bower_components/emby-webcomponents/sessionplayer");
+            (browser.chrome || browser.opera) && list.push("bower_components/emby-webcomponents/chromecast/chromecastplayer");
```
Comment (JustAMan) [inline src/scripts/site.js]:
```suggestion
            if (browser.chrome || browser.opera) {
                list.push("bower_components/emby-webcomponents/chromecast/chromecastplayer");
            }
```

File: src/scripts/site.js [hunk 4]
```diff
@@ -870,8 +870,20 @@ var Dashboard = {
...
-        "android" === self.appMode ? self.MainActivity && MainActivity.enableVlcPlayer() && list.push("cordova/vlcplayer") : "cordova" === self.appMode && (list.push("cordova/audioplayer"), (browser.iOSVersion || 0) >= 11 && list.push("cordova/mpvplayer")), list.push("bower_components/emby-webcomponents/htmlaudioplayer/plugin"), "cordova" === self.appMode && list.push("cordova/chromecast"), "android" === self.appMode && list.push("cordova/externalplayer"), list.push("bower_components/emby-webcomponents/htmlvideoplayer/plugin"), list.push("bower_components/emby-webcomponents/photoplayer/plugin"), appHost.supports("remotecontrol") && (list.push("bower_components/emby-webcomponents/sessionplayer"), (browser.chrome || browser.opera) && list.push("bower_components/emby-webcomponents/chromecast/chromecastplayer")), list.push("bower_components/emby-webcomponents/youtubeplayer/plugin");
+        var list = [];
+        list.push("bower_components/emby-webcomponents/playback/playbackvalidation");
```
[the remark this answers is not in the document]
Comment (dkanada) [reply src/scripts/site.js]: Most of them weren't before, there were only three in the inititalization before I deminified, but I thought this looked nicer so I left it.

**Pull request merged**